### ISSUE TITLE
Using this for creating backpropagation saliency maps?

### ISSUE THREAD

**Issue by greglahmer** [MECLabTUDA/M3d-Cam#25]
If we are trying to establish, just the pixels instead of image regions with the highest gradients attached to them, I have heard saliency maps using backpropagation is the best way? Is it possible to do this with medcam as well?

**Comment by Karol-G** [comment]
You probably mean Grad-CAM? This is the default setting of medcam.

**Comment by stevenagl12** [comment]
I think they may be referring to saliency maps that calculate the weight of the pixels or voxels in an image, not the gradients. Maybe something like: https://towardsdatascience.com/saliency-map-using-pytorch-68270fe45e80.

**Comment by greglahmer** [comment]
Yes, that is what I meant.
